tests/platform_tests/api/test_sfp.py

```diff
@@ -477,6 +477,14 @@ def test_power_override(self, duthosts, enum_rand_one_per_hwsku_hostname, localh
                 self.expect(power_override is False, "Transceiver {} power override data is incorrect".format(i))
         self.assert_expectations()
 
+    def test_get_error_description(self, duthosts, enum_rand_one_per_hwsku_hostname, localhost, platform_api_conn):
+        for i in self.candidate_sfp:
```

Comment:
can you please add a description of what this tests stands for and what is the expected behaviour?

Reply:
Fixed